kernel/data-pipeline/distsql/handler/src/main/java/org/apache/shardingsphere/migration/distsql/handler/query/ShowMigrationCheckStatusResultSet.java

```diff
@@ -42,13 +43,17 @@ public final class ShowMigrationCheckStatusResultSet implements DatabaseDistSQLR
     @Override
     public void init(final ShardingSphereDatabase database, final SQLStatement sqlStatement) {
         ShowMigrationCheckStatusStatement checkMigrationStatement = (ShowMigrationCheckStatusStatement) sqlStatement;
-        ConsistencyCheckJobItemInfo info = jobAPI.getJobItemInfo(checkMigrationStatement.getJobId());
-        String checkResult = null == info.getCheckSuccess() ? "" : info.getCheckSuccess().toString();
-        Collection<Object> result = Arrays.asList(Optional.ofNullable(info.getTableNames()).orElse(""), checkResult,
-                String.valueOf(info.getFinishedPercentage()), info.getRemainingSeconds(),
-                Optional.ofNullable(info.getCheckBeginTime()).orElse(""), Optional.ofNullable(info.getCheckEndTime()).orElse(""),
-                info.getDurationSeconds(), Optional.ofNullable(info.getErrorMessage()).orElse(""));
-        data = Collections.singletonList(result).iterator();
```

Comment:
This code block could be extracted as a new method, and then reference it in for each loop